infrastructure/permissions.tf

```diff
@@ -216,3 +216,32 @@ resource "aws_iam_role_policy_attachment" "cloudwatch" {
   role = "${aws_iam_role.data_refinery_instance.name}"
   policy_arn = "${aws_iam_policy.cloudwatch_policy.arn}"
 }
+
+resource "aws_s3_bucket_policy" "cloudtrail_access_policy" {
+  bucket = "${aws_s3_bucket.data_refinery_bucket.id}"
```

Comment:
This also will need to change to reference the same logging bucket as `aws_cloudtrail.data_refinery_s3_cloudtrail`

Reply:
👍